Build release GH Action is broken: No rule to make target '/pato-to-qudt-quantitykind.sssom.tsv', needed by 'all_assets'.  Stop.

When running the Build release files GH Action, it fails with the error message

```bash
make[1]: *** No rule to make target '/pato-to-qudt-quantitykind.sssom.tsv', needed by 'all_assets'.  Stop.
make[1]: Leaving directory '/__w/pato/pato/src/ontology'
make: *** [Makefile:313: prepare_release_fast] Error 2
Error: Process completed with exit code 2.
```

--- a/docs/odk-workflows/ManageDocumentation.md
+++ b/docs/odk-workflows/ManageDocumentation.md
@@ -43,5 +43,3 @@ The documentation is _not_ automatically updated from the Markdown, and needs to
 3. Just to double check, you can now navigate to your documentation pages (usually https://pato-ontology.github.io/pato/). 
    Just make sure you give GitHub 2-5 minutes to build the pages!
 
-
-

--- a/src/ontology/Makefile
+++ b/src/ontology/Makefile
@@ -10,7 +10,7 @@
 # More information: https://github.com/INCATools/ontology-development-kit/
 
 # Fingerprint of the configuration file when this Makefile was last generated
-CONFIG_HASH=                8c38f556f1076ce2a46088cd083451de27b8553932affe92d72c22f02b6e92e4
+CONFIG_HASH=                97037c2418df585a2332da8a26b11ac73e9ae6c3bb97d1371443b254249a7ab1
 
 
 # ----------------------------------------
@@ -57,6 +57,12 @@ ANNOTATE_CONVERT_FILE =     annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONT
 OTHER_SRC =                 $(COMPONENTSDIR)/pato_ext.owl 
 ONTOLOGYTERMS =             $(TMPDIR)/ontologyterms.txt
 EDIT_PREPROCESSED =         $(TMPDIR)/$(ONT)-preprocess.owl
+MAPPINGDIR=                 ../mappings
+MAPPING_TESTER=             sssom validate
+SSSOMPY=                    sssom
+MAPPINGS=                   pato-to-qudt-quantitykind 
+MAPPING_RELEASE_FILES=      $(foreach n,$(MAPPINGS), $(MAPPINGDIR)/$(n).sssom.tsv)
+
 
 FORMATS = $(sort  owl obo json owl)
 FORMATS_INCL_TSV = $(sort $(FORMATS) tsv)
@@ -570,6 +576,22 @@ custom_reports: $(EDIT_PREPROCESSED) | $(REPORTDIR)
 ifneq ($(SPARQL_EXPORTS_ARGS),)
 	$(ROBOT) query -f tsv --use-graphs true -i $< $(SPARQL_EXPORTS_ARGS)
 endif
+# ----------------------------------------
+# SSSOM Mapping Files
+# ----------------------------------------
+
+validate-sssom-%:
+	tsvalid $(MAPPINGDIR)/$*.sssom.tsv --comment "#"
+	sssom validate $(MAPPINGDIR)/$*.sssom.tsv
+
+# This mappingset is manually curated, so we only check that the file actually exists.
+$(MAPPINGDIR)/pato-to-qudt-quantitykind.sssom.tsv:
+	test -f $@
+
+validate_mappings:
+	$(MAKE_FAST) $(foreach n,$(MAPPINGS), validate-sssom-$(n))
+
+
 
 # ----------------------------------------
 # Release artefacts: export formats

--- a/src/ontology/pato-odk.yaml
+++ b/src/ontology/pato-odk.yaml
@@ -5,6 +5,7 @@ git_main_branch: master
 repo: pato
 report_fail_on: none
 edit_format: obo
+use_mappings: True
 export_formats:
   - owl
   - obo

